src/cc_catalog_airflow/dags/util/test_operator_util.py

```diff
@@ -0,0 +1,17 @@
+import os
+import sys
+from airflow import DAG
+from datetime import datetime
+import util.operator_util as o
```

Comment:
I tend toward pretty short aliases in test files, since properly named test functions and test files themselves should make where the function was imported from obvious.  I'll change it regardless.